src/core/tests/engines_util/CMakeLists.txt

```diff
@@ -8,9 +8,9 @@ add_library(engines_test_util STATIC EXCLUDE_FROM_ALL ${ENGINES_UTIL_SRC})
 
 ie_faster_build(engines_test_util UNITY)
 
-target_include_directories(engines_test_util PRIVATE $<TARGET_PROPERTY:inference_engine,INTERFACE_INCLUDE_DIRECTORIES>)
-target_include_directories(engines_test_util PRIVATE $<TARGET_PROPERTY:inference_engine_transformations,INTERFACE_INCLUDE_DIRECTORIES>)
-target_link_libraries(engines_test_util PUBLIC ngraph inference_engine inference_engine_transformations gtest gmock ngraph_test_util)
+target_include_directories(engines_test_util PRIVATE $<TARGET_PROPERTY:openvino::runtime,INTERFACE_INCLUDE_DIRECTORIES>)
+target_include_directories(engines_test_util PRIVATE $<TARGET_PROPERTY:openvino::runtime::dev,INTERFACE_INCLUDE_DIRECTORIES>)
```

Comment:
Why do we need `target_include_directories` while we have `target_link_libraries` below?